gpkit/constraints/base.py

```diff
@@ -0,0 +1,54 @@
+"Implements BaseConstraint"
+from collections import defaultdict, Iterable
+from .set import ConstraintSet
+from ..varkey import VarKey
+from ..nomials import Variable, Signomial
+from .. import SignomialsEnabled
+
+
+class ConstraintBase(ConstraintSet):
+    "A ConstraintSet for making named models with setup methods"
```

Comment:
I know, I still believe in the backwards compatibility goal.

Sorry for dumb questions - but what does Model do that ConstraintBase can't, or vice versa?